nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -25,7 +25,7 @@ if (params.help) {
 /*
  * Stage config files
  */
-ch_multiqc_config = file("$baseDir/assets/multiqc_config.yaml", checkIfExists: true)
+ch_multiqc_config = Channel.fromPath("$baseDir/assets/multiqc_config.yaml", checkIfExists: true)
```

Comment:
Why did you have to change this? Staging issues?

Reply:
```suggestion
ch_multiqc_config = file("$baseDir/assets/multiqc_config.yaml", checkIfExists: true)
```

Reply:
Can change this back with the other change I made below.

Reply:
That also works, right. But using `file()` does not generate a channel but just a path. Then one cannot, e.g. print the content with `.dump`. Is this preferred? 
The following line `ch_multiqc_custom_config` also defines a channel with `Channel.fromPath()`. Maybe we can decide on which one is the preferred one and I'll change them all :)